config/zulu_jre.yml

```diff
@@ -0,0 +1,44 @@
+# Cloud Foundry Java Buildpack
+# Copyright 2016 the original author or authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Configuration for JRE repositories keyed by vendor
+# Pre Java 1.8, permgen was used instead of metaspace.  Please see the documentation for more detail.
+
+# You must specify a the repository root of an Zulu JRE repository. Please see the documentation for more detail.
+# e.g.  repository_root: "http://example.com/zulu-jre/{platform}/{architecture}"
+---
+jre:
+  version: 1.8.0_+
+  repository_root: "http://cdn.azul.com/zulu/bin"
```

Comment:
@Ekaterina001 Is there an `https` form of this repository?  We favor SSL connections wherever possible.